integration_tests/e2e/goal/addStep.cy.ts

```diff
@@ -128,6 +129,6 @@ context('Add a step', () => {
     addStepPage.submitPage()
 
     // Then
-    // assert we are on the next page
+    Page.verifyOnPage(AddNotePage)
```

Comment:
Do we need to enable this test? (Line 111 ?)
Or; does the test not really add anything over and above the test in `addNote` above?, which is the 'screen to screen journey' ?